.github/workflows/tests.yml

```diff
@@ -50,12 +50,14 @@ jobs:
 
```

Comment:
Good point. This is implicitly covered by the different python versions, because older python versions do not support 0.7.19, while the new ones do pull the latest. I think we can move on as is

Reply:
That is good to know! I'd have maybe added a comment in test.yaml about that or something, for auditing purposes.
Thank you for fixing! I only added my reviews since it is OS and I wanted to share some of the burden, I really appreciate this package.